src/Bridge/Doctrine/Orm/AbstractPaginator.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Doctrine\Orm;
+
+use ApiPlatform\Core\DataProvider\PartialPaginatorInterface;
+use Doctrine\ORM\Tools\Pagination\Paginator;
+
+abstract class AbstractPaginator implements \IteratorAggregate, PartialPaginatorInterface
```

Comment:
FYI,

`abstract class AbstractPaginator implements PartialPaginatorInterface, \IteratorAggregate` => fatal error
`abstract class AbstractPaginator implements \IteratorAggregate, PartialPaginatorInterface` => it works

🤔 🤣